src/lib/CofEnv.mli

```diff
@@ -24,6 +24,6 @@ val test_sequent : env -> D.cof list -> D.cof -> bool
 (** Monadic interface *)
 module M (M : CoolBasis.Monad.S) :
 sig
-    (* Search all branches induced by unreduced joins under additional cofibrations. *)
-    val left_invert_under_cofs : env -> D.cof list -> (env -> unit M.m) -> unit M.m
+  (* Search all branches induced by unreduced joins under additional cofibrations. *)
+  val left_invert_under_cofs : env -> D.cof list -> (env -> unit M.m) -> unit M.m
 end
```

Comment:
Do extra space characters come from the sky?

Reply:
That's it!!!